File: Unidad I/PAC II 2019/Codigos/1-LinkedList/Python/LinkedList.py
#-*- coding.utf8 -*-
class Node: 
    def __init__(self, name, student):
        self.name = name
        self.student = student
        self.next = None

class LinkedList:
    def __init__(self, Node):
        self.first = Node

class Student:
    def __init__(self, name, accountNumber):
        self.name = name
        self.accountNumber = accountNumber
        self.average = "100"

Juan = Student("Juan", "20193012")
node = Node(0, Juan)
listS = LinkedList(node)

print("la variable lista es: %s" % listS)
print("la variable nodo es: %s" % node)
print("la variable estudiante es: %s" % Juan)

File: Unidad I/PAC II 2019/Codigos/1-LinkedList/JS/LinkedList.js
function Node(name, student) {
    this.name = name;
    this.student = student;
    this.next = null;
}

function LinkedList(Node) {
    this.first = Node;
}

function Student(name, accountNumber) {
    this.name = name;
    this.accountNumber = accountNumber;
    this.average = 100;
}

Juan = new Student("Juan", "2010293");
Node = new Node(0, Juan);
List = new LinkedList(Node);